prdoc/pr_9480.prdoc

```diff
@@ -0,0 +1,85 @@
+title: Lazy decode block extrinsics
+doc:
+- audience: Runtime Dev
+  description: |-
+    Related to https://github.com/paritytech/polkadot-sdk/issues/4255
+
+    Addresses https://github.com/paritytech/polkadot-sdk/pull/7902#pullrequestreview-3079855302
+
+    This PR enables us to lazily decode the block extrinsics when needed.
+crates:
+- name: sp-runtime
+  bump: major
+- name: frame-executive
+  bump: major
+- name: cumulus-primitives-core
+  bump: minor
+- name: pallet-revive
+  bump: minor
+- name: frame-benchmarking-cli
+  bump: patch
+- name: sc-rpc
+  bump: patch
+- name: sc-service
+  bump: patch
+- name: sc-tracing
+  bump: patch
+- name: sp-api
+  bump: patch
+- name: cumulus-client-consensus-aura
+  bump: patch
+- name: cumulus-pallet-parachain-system
+  bump: patch
+- name: polkadot-service
+  bump: patch
+- name: sc-basic-authorship
+  bump: patch
+- name: frame-support
+  bump: patch
+- name: sp-api-proc-macro
+  bump: patch
+- name: frame-try-runtime
+  bump: patch
+- name: sp-block-builder
+  bump: patch
+- name: frame-support-procedural
+  bump: patch
```

Comment:
I would mark  `sp-api-proc-macro`, `frame-support-procedural` as major because they don't directly depend on the crate they use so tools miss them when publishing new version.
```suggestion
- name: sp-api
  bump: major
- name: cumulus-client-consensus-aura
  bump: patch
- name: cumulus-pallet-parachain-system
  bump: patch
- name: polkadot-service
  bump: patch
- name: sc-basic-authorship
  bump: patch
- name: frame-support
  bump: major
- name: sp-api-proc-macro
  bump: major
- name: frame-try-runtime
  bump: patch
- name: sp-block-builder
  bump: patch
- name: frame-support-procedural
  bump: major
```

Reply:
Done